data_policy.md

```diff
@@ -49,12 +50,14 @@
 
 3.2 Curation of raw data and associated metadata
 
-3.2.1 All raw data will be created in well-defined formats, for which the means of reading the data will be made available by the Facility.
+3.2.1 All raw data will be created in standardized formats, for which the means of reading the data will be made available by the Facility.
```

Comment:
This might be overkill, but was wondering if we want to be more specific about the formats being interoperable. Maybe something like:

```suggestion
3.2.1 All raw data will be created in formats with publicly available standards, for which the means of reading the data will be made available by the Facility.
```

Reply:
Good thought, but I would argue not, because what is meant by 'publicly available' here... Is the old .raw ISIS bespoke format (historically and still in use for some instrument along size nexus) a publicly available standards - the answer to this will depend on who you ask. However, the old .raw ISIS is a standardized format, that is a standard for storing ISIS data, which I think fewer people can disagree with (similar argument can also be used for the bespoke ISIS nexus ways of storing instrument data!)